packages/expo-image/android/src/main/java/com/caverock/androidsvg/SVGStyler.kt

```diff
@@ -19,15 +23,83 @@ internal fun replaceStyles(style: SVG.Style?, newColor: Int) {
   replaceColor(style.viewportFill, newColor)
 }
 
-internal fun applyTintColor(element: SVG.SvgObject, newColor: Int) {
-  if (element is SVG.SvgElementBase) {
+internal fun hasStyle(element: SvgElementBase): Boolean {
+  if (element.style == null && element.baseStyle == null) {
+    return false
+  }
+
+  val style = element.style
+  val hasColorInStyle = style != null &&
+    (
+      style.color != null || style.fill != null || style.stroke != null ||
+        style.stroke != null || style.stopColor != null || style.solidColor != null
+      )
```

Comment:
What a weird formatting at first glance 🙈

Reply:
Yes, it's weird, but it seems like it's from the Kotlin default code style